fgd/point/logic/logic_playmovie.fgd

```diff
@@ -2,7 +2,70 @@
 	appliesto(since_ASW) 
 	iconsprite("editor/logic_playmovie") = logic_playmovie: "Plays a movie and allows for various playback options"
 	[
-	moviefilename(string) : "Movie to play" : : "Filename of the movie to play"
+	MovieFilename(choices) : "Movie to display" : "aperture_logo.bik" : "The movie to display. You can also type in the filename of a custom movie here." =
```

Comment:
For this, the way you can do this is to have a bunch of duplicate keyvalues with different tags:

```suggestion
	MovieFilename(string) : "Movie to display" : "" : "The movie to display from the media/ folder. The BIK file extension must be included."
	
	MovieFilename[P2](choices) : "Movie to display" : "aperture_logo.bik" : "The movie to display. You can also type in the filename of a custom movie that is present in the media/ folder. The BIK file extension must be included." =
```

This way it provides movie names when we know them, and falls back to a string for other games.